### PR TITLE
Extract rogue env vars

### PR DESCRIPTION
**Pull request by dragonchaser**
We have added a little tool that will extract global env vars, that are loaded
only through os.Getenv for documentation purposes

refs #4916

### CODE REVIEW

File: docs/templates/ADOC_global.tmpl [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+a| `{{- .Name }}` +
+a| [subs=-attributes]
+++{{ .Type }} ++
+a| [subs=-attributes]
+++{{.DefaultValue}} ++
+a| [subs=-attributes]
+{{.Description}}
+
+{{- end }}
+| === |
```
Comment (mmattel) [inline docs/templates/ADOC_global.tmpl:23]:
```suggestion
[caption=]
.Environment variables with global scope
[width="100%",cols="~,~,~,~",options="header"]
|===
| Name
| Type
| Default Value
| Description

{{- range .Variables}}

a| `{{- .Name }}` +
a| [subs=-attributes]
++{{ .Type }} ++
a| [subs=-attributes]
++{{.DefaultValue}} ++
a| [subs=-attributes]
{{.Description}}

{{- end }}
|===
```
Your editor is consequently adding stuff we afterwards need to remove 🤣

File: docs/templates/ADOC_global.tmpl [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[caption=]
+.Environment variables with global scope
```
Comment (mmattel) [inline docs/templates/ADOC_global.tmpl:2]:
```suggestion
// collected from .getenv

[caption=]
.Environment variables with global scope not included in a service
```

**Comment by mmattel** [comment]
When running locally, regarding `global_vars.yaml`:
* some entries are getting double present like `MICRO_LOG_LEVEL` (but not only)
`path: ocis-pkg/log/log.go:34`
`path: ocis-pkg/log/log.go:30`
* strange env vars like (rawname):
`parts[0]`
`registryAddressEnv`
* description text
When adding a description text and re-running `make docs-generate`, there is no change in `global_configvars.adoc`

Because of the double present entries, this is confusing as where to add description text necessary for the adoc table output.

**Review by mmattel — APPROVED** [review]

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiGraph-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/17471/47/1) failed. Further test are cancelled...

**Pull request merged**